internal/repository/transaction_repo.go

```diff
@@ -10,11 +10,13 @@ import (
 
 var (
 	ErrTransactionNotFound = errors.New("transaction not found")
-	//ErrMissingParams   = errors.New("must provide a name and last name for an account")
+	ErrMissingFields       = errors.New("transaction is missing one or more of these: owner, sender, receiver, amount")
```

Comment:
we can make the errors more specific by returning something like on your function
```go
errors.Wrap(ErrMissingFields, "sender")
```